assets/erc-7654/RequestMethodTypes.sol

```diff
@@ -0,0 +1,95 @@
+// SPDX-License-Identifier: GPL-3.0
```

Comment:
We cannot allow GPL code in reference implementations. That would mean anyone writing code based on your example would also need to be GPL.

If you have the permission of the copyright holder, please relicense this under CC0-1.0 (preferably) or something like MIT or Apache.